docker/dockerfile-build-ubuntu-rock

```diff
@@ -18,8 +18,6 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     build-essential \
     ca-certificates \
     git \
-    gcc \
-    g++ \
     make \
     cmake \
     clang-format-3.8 \
```

Comment:
do we need to remove clang-format as well?

Reply:
guess we can remove it, but shouldnt break anything

Reply:
removing that now